### ISSUE TITLE
Technomancer and healing

### ISSUE THREAD

**Issue by SilverTalismen** [PolarisSS13/Polaris#2668]
#### Brief description of the issue
None of the healing spells close incisions, not even healing from death
#### What you expected to happen
To revive someone and incisions to close
#### What actually happened
Revived them, they promptly bled out again
#### Steps to reproduce
make a shit ton of incisions on dead techno, have omeone revive and heal, watch incisions stay
#### Additional info:
 - **Server Revision**: master - 2016-10-21
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by FenodyreeAv** [comment]
Admin rejuv also did nothing to help, required a mob being spawned in to individually cauterize every single one.

**Comment by Anewbe** [comment]
Yeah, this gon be a big handful of changes, I'll get to it sooner or later.

**Comment by Anewbe** [comment]
I can't actually reproduce this issue, we'll keep an eye out for it in the future.
